test/worker/python/sensing/single-target-coloristouchingcolor.py

```diff
@@ -0,0 +1,6 @@
+async def thread_id_0(vm_proxy):
+    isColorTouchingColor = vm_proxy.isColorTouchingColor
+    say = vm_proxy.say
+    result = await isColorTouchingColor('green', 'red')
+    say(result)
```

Comment:
```suggestion
    await say(result)
```
say needs to be awaited here to match the output of the linker

Reply:
The worker tests all passed without awaiting the say blocks (they also work with the await say()), I just pushed the change to await the says but I'll re-PR tomorrow